Makefile

```
@@ -56,7 +56,7 @@ docs: docs/api/openapi3.json
 
 .PHONY: docs/api/openapi3.json
 docs/api/openapi3.json:
-	go run ./internal/openapigen $@
+	go test ./internal/server -run TestWriteOpenAPIDocToFile -update
```

Another option that just came to mind is:

```suggestion
	go run -ldflags '-s -X github.com/infrahq/infra/internal.Version=0.0.0' ./internal/openapigen $@
```

I think this allows us to check that `internal/openapigen` still works ahead of a release. I tried it locally and it seems to generate the correct doc.